pkg/vsphere/datastore/datastore.go

```diff
@@ -36,6 +36,15 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/tasks"
 )
 
+// All paths on the datastore for images are relative to <datastore>/VIC/
+var StorageParentDir = "VIC"
```

Comment:
There is an explicit `lib/constants` package for sharing this kind of shared constant.
The specifics like this don't really belong in the supposedly generic `pkg` implementations.

Can you move the constants under `lib/constants` and update the references please.

Reply:
sounds good to me.